ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -160,18 +160,15 @@ else
 	#
 	# Set AUTOCNF flag to use it in AUTOSTART scripts.
 	#
-	if param compare SYS_AUTOCONFIG 1
+	if param greater SYS_AUTOCONFIG 0
 	then
-		# Wipe out params except RC*, flight modes, total flight time, accel cal, gyro cal
-		param reset_nostart RC* COM_FLTMODE* LND_FLIGHT_T_* TC_* CAL_ACC* CAL_GYRO*
-		set AUTOCNF yes
-	else
-		if param compare SYS_AUTOCONFIG 2
+		if param compare SYS_AUTOCONFIG 1
 		then
-			set AUTOCNF yes
-		else
-			set AUTOCNF no
```

Comment:
I don't see this set to `no` in the else case anymore. If not set, it could cause problems with script evaluation.

Reply:
Thanks for catching that. That deletion shouldn't have occurred. The unset remains at the bottom of the script, but the script still lacks a single design pattern in that regard.  By the logic we've discussed above the set/unset for AUTOCNF should probably be moved to this location, but then we'd slowly end up where we were last year with variables not getting unset.  My preference would still be a single rule to follow.